ui/apps/ui/src/app/components/sort-by-functionality/sort-option-values.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum SortOptionValuesEnum {
+  RELEVANCY = 'Relevancy',
```

Comment:
If your defining user readable text you probably should use `$localize`. But I do not think that storing captions for end user in the enum is the best idea.

Reply:
i18n labels added